charts/rollout-operator/templates/deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
-        - name: {{ .Chart.Name }}
+        - name: rollout-operator
```

Comment:
```suggestion
        - name: {{ include "rollout-operator.chartName" . }}
```

Reply:
This was intentional , otherwise it's hard to write dashboards , see commit message https://github.com/grafana/helm-charts/pull/1275/commits/81c5656730273c261233a20c13746d6239ec1fff